src/main/kotlin/net/lumalyte/lg/interaction/menus/guild/GuildWarAcceptanceMenu.kt

```
@@ -133,13 +133,13 @@ class GuildWarAcceptanceMenu(
         val bannerItem = targetGuild.banner?.let { banner ->
             try {
                 val deserialized = banner.deserializeToItemStack()
-                deserialized?.clone() ?: ItemStack(Material.WHITE_BANNER)
+                deserialized?.clone() ?: ItemStack.of(Material.WHITE_BANNER)
             } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)